src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -1140,7 +1150,8 @@ public IEnumerable<(string Key, long Index)> GetCompareExchangeIndexes(Transacti
         private static unsafe string ReadCompareExchangeKey(TableValueReader reader, string dbPrefix)
         {
             var ptr = reader.Read((int)UniqueItems.Key, out var size);
-            return Encodings.Utf8.GetString(ptr, size).Substring(dbPrefix.Length + 4);
+            // we need to read only the key from the format: 'databaseName/key'
+            return Encodings.Utf8.GetString(ptr, size).Substring(dbPrefix.Length + 1);
```

Comment:
why this change now?

Reply:
Because we changed the key format. (removed the 'db/' prefix)